src/utils/functions/identifications/identificationFunctions.ts

```diff
@@ -32,35 +34,44 @@ export type IdentificationQuestionValue = {
   text: string;
   options: IdentificationQuestionOption[];
   dependsOn?: { questionId: IdentificationQuestionsId; values: string[] };
+  disabled?: boolean;
 };
 
 export type IdentificationQuestions = Partial<
   Record<IdentificationQuestionsId, IdentificationQuestionValue>
 >;
 
-export const identificationQuestionsTree: Record<
-  IdentificationConfiguration,
-  IdentificationQuestions
-> = {
-  [IdentificationConfiguration.INDTEL]: indtelIdentificationQuestionsTree,
-  [IdentificationConfiguration.IASCO]: houseF2FIdentificationQuestionsTree,
-  [IdentificationConfiguration.NOIDENT]: {},
-  [IdentificationConfiguration.HOUSEF2F]: houseF2FIdentificationQuestionsTree,
-  [IdentificationConfiguration.HOUSETEL]: houseTelIdentificationQuestionsTree,
-  [IdentificationConfiguration.HOUSETELWSR]: houseTelIdentificationQuestionsTree,
-  [IdentificationConfiguration.INDTELNOR]: indtelIdentificationQuestionsTree,
-  [IdentificationConfiguration.INDF2F]: {},
-  [IdentificationConfiguration.SRCVREINT]: {},
+export const identificationQuestionsTree = (
```

Comment:
suggestion: evolved from an Object to a function, so maybe rename it to getIdentificationQuestionsTree